wiki/osu!supporter/en.md

```diff
@@ -2,38 +2,38 @@
 
 ![osu!supporter tag](/wiki/shared/osu!supporter.png "osu!supporter tag")
 
-![osu!supporter link on main page](Osu-support.png " osu!supporter link on main page.")
+![osu!supporter link on main page](./Osu-support.png " osu!supporter link on main page.")
 
 Refer to link: <https://osu.ppy.sh/p/support>
```

Comment:
eh, don't use this markup for links
`[Supporting osu!](https://osu.ppy.sh/p/support)`